lms/templates/instructor/instructor_dashboard_2/instructor_dashboard_2.html

```diff
@@ -146,6 +146,19 @@ <h3 class="hd hd-3" id="header-${ section_data['section_key'] }">${ section_data
   </div>
 </div>
 
+<%static:optional_include_mako file="google_analytics.html" is_theming_enabled="True" />
+<% ga_acct = static.get_value("GOOGLE_ANALYTICS_ACCOUNT", settings.GOOGLE_ANALYTICS_ACCOUNT) %>
+% if ga_acct:
+  <script type="text/javascript>
+    jQuery(document).ready(function () {
+      jQuery('.menu a').click(function(){
+        var match = jQuery(this).attr('href').match(/#\S+/);
```

Comment:
Assuming we update the event binding selectors above,
I think we can simplify  this to just be 
`ga('send', 'pageview', { 'page': location.pathname + location.search + location.hash});`

like we're doing elsewhere.
That way, we're using the same form of `ga('send', ...)` everywhere.
We also shouldn't need the regex if we're selecting the proper links, right?

Let me know if I'm overlooking anything..

Reply:
actually, @stvstnfrd, I THINK this is meant to send when links are clicked, regardless of pageload?

Reply:
@caesar2164 I don't follow; I'm not sure i've said this _isn't_ about clicks?

Reply:
right, but I don't think the location is updated yet at this point...

Reply:
Ah, right; disregard that comment.